MetaMask/metamask-mobile
# feat(2796): permission settings replace some of the mock data by real data

## **Description**
The data is not ready yet to display permissions of all source, it will require a 'source' attribute to say if its sdk, wallet connect or in app browser. This is an attempt at filtering them based on differences the 'might' always have, but even if it did work now, it could break anytime because it requires a real source property meant to identify the source.


## **Related issues**

Fixes:

## **Manual testing steps**

1. set the feature flag export MM_MULTICHAIN_V1_ENABLED="1"
2. go to settings -> permissions
3. if you have in app browser permissions, they should be visible in this list of permissions

## **Screenshots/Recordings**



| Before (mocked list)       | After  (not mocked list)     |
|--------------|--------------|
| <img width="350" alt="Screenshot 2024-04-18 at 3 56 43 PM" src="https://github.com/user-attachments/assets/2a46f217-742a-4651-a7d0-71c2cef60fde"> |<img width="350" alt="Screenshot 2024-04-18 at 3 56 43 PM" src="https://github.com/user-attachments/assets/1f965baa-89c8-4745-b378-1281c7001db2"> |



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
